### ISSUE TITLE
python-amulet dependency not in PPA

### ISSUE THREAD

**Issue by sparkiegeek** [juju/amulet#118]
On Xenial, after adding ppa:juju/stable when I try to install python-amulet I get:

> Reading package lists... Done
> Building dependency tree       
> Reading state information... Done
> Some packages could not be installed. This may mean that you have
> requested an impossible situation or if you are using the unstable
> distribution that some required packages have not yet been created
> or been moved out of Incoming.
> The following information may help to resolve the situation:
> 
> The following packages have unmet dependencies.
>  python-amulet : Depends: python-libcharmstore but it is not installable
> E: Unable to correct problems, you have held broken packages.

It wasn't clear where to file this, but @mitechie suggested to start here

**Comment by sparkiegeek** [comment]
Note that now the python-path issue has been resolved, `python3-amulet` **is** installable

**Comment by mitechie** [comment]
@sparkiegeek where was the python-path issue? What was the fix there? Was it a xenial issue that is updated or a packaging/python issue you manually corrected?

**Comment by marcoceppi** [comment]
This is being addressed

**Comment by sparkiegeek** [comment]
@mitechie yes, it's in https://bugs.launchpad.net/amulet/+bug/1554028

**Comment by ajkavanagh** [comment]
This seems to be fixed now (thanks!).  I've successfully installed amulet on xenial now.

**Comment by johnsca** [comment]
This seems to still be an issue on trusty:

```
ubuntu@murdoch:~$ sudo apt-get update
Ign http://archive.ubuntu.com trusty InRelease
Hit http://ppa.launchpad.net trusty InRelease
Hit http://archive.ubuntu.com trusty-updates InRelease
Hit http://archive.ubuntu.com trusty-security InRelease
Hit http://archive.ubuntu.com trusty Release.gpg  
Hit http://ppa.launchpad.net trusty/main amd64 Packages                 
Hit http://archive.ubuntu.com trusty Release                            
Hit http://archive.ubuntu.com trusty-updates/main Sources
Hit http://archive.ubuntu.com trusty-updates/restricted Sources
Hit http://archive.ubuntu.com trusty-updates/universe Sources
Hit http://archive.ubuntu.com trusty-updates/main amd64 Packages
Hit http://archive.ubuntu.com trusty-updates/restricted amd64 Packages
Hit http://archive.ubuntu.com trusty-updates/universe amd64 Packages
Hit http://archive.ubuntu.com trusty-security/main Sources
Hit http://archive.ubuntu.com trusty-security/restricted Sources
Hit http://archive.ubuntu.com trusty-security/universe Sources
Hit http://archive.ubuntu.com trusty-security/main amd64 Packages
Hit http://archive.ubuntu.com trusty-security/restricted amd64 Packages
Hit http://archive.ubuntu.com trusty-security/universe amd64 Packages
Hit http://archive.ubuntu.com trusty/main Sources
Hit http://archive.ubuntu.com trusty/restricted Sources
Hit http://archive.ubuntu.com trusty/universe Sources
Hit http://archive.ubuntu.com trusty/main amd64 Packages
Hit http://archive.ubuntu.com trusty/restricted amd64 Packages
Hit http://archive.ubuntu.com trusty/universe amd64 Packages
Reading package lists... Done
ubuntu@murdoch:~$ sudo apt-get install amulet
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 amulet : Depends: python-amulet (> 1.12.0) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by marcoceppi** [comment]
This has been resolved with 1.14.0 of Amulet
